app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__TargetDetailsEditor.re

```diff
@@ -580,7 +580,7 @@ let quizEditor = (state, send) => {
 };
 
 let saveDisabled = state => {
-  let hasValidTitle = state.title |> String.length > 1;
+  let hasValidTitle = state.title |> String.trim |> String.length > 1;
```

Comment:
There is a mismatch between the validation here, which decides whether the save button should be disabled, and the validation which decides to show the error message (Line 746).

However, [the server-side validation](https://github.com/SVdotCO/pupilfirst/blob/ee37f02a71916ff2b63d13a3612d2403f96558e1/app/queries/update_target_mutator.rb#L5) only requires a minimum length of _one_ on a target's title.

So _this_ check should be `> 0`, and the one in `School__InputGroupError` should be `< 1`.

Reply:
Thank you. Have made the changes.

`School_InputGroupError` did have a `< 1` before as well. Hence, I've just trimmed it.